contracts/0.8.9/DepositSecurityModule.sol

```diff
@@ -160,6 +160,7 @@ contract DepositSecurityModule {
     }
 
     function _setPauseIntentValidityPeriodBlocks(uint256 newValue) internal {
+        require(newValue > 0, "invalid value");
```

Comment:
Maybe change to something readable? Like "pauseIntentValidityPeriodBlocks should be greater than zero"?
WDYT?